<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Jméno</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Pozice</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineJmeno"/>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="comboPost">
     <item>
      <property name="text">
       <string>Lavička</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SDZ</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SOZ</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LZ</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PZ</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SÚ</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LK</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PK</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HÚ</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Věk</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="spinBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
